api/api_ethereum.go

```diff
@@ -1453,7 +1453,45 @@ func (api *EthereumAPI) PendingTransactions() ([]*EthRPCTransaction, error) {
 // Resend accepts an existing transaction and a new gas price and limit. It will remove
 // the given transaction from the pool and reinsert it with the new gas price and limit.
 func (api *EthereumAPI) Resend(ctx context.Context, sendArgs EthTransactionArgs, gasPrice *hexutil.Big, gasLimit *hexutil.Uint64) (common.Hash, error) {
-	return common.Hash{}, errors.New("this api is not supported by Klaytn because Klaytn use fixed gasPrice policy")
+	if sendArgs.Nonce == nil {
```

Comment:
```go
func klaytnTypedArgs(sendArgs EthTransactionArgs) SendTxArgs {
	ethTx := sendArgs.toTransaction()
	txType := ethTx.Type()
	return SendTxArgs{
		TypeInt: &txType,
		From: *sendArgs.From,
		Recipient: *&sendArgs.To,
		...
	}
}

func (api *EthereumAPI) Resend(ctx context.Context, sendArgs EthTransactionArgs, gasPrice *hexutil.Big, gasLimit *hexutil.Uint64) (common.Hash, error) {
	sendTxArgs := klaytnTypedArgs(sendArgs)
	return api.publicTransactionPoolAPI.Resend(sendTxArgs)
}
```
How do you think about this?

Reply:
Or you can make an interface for EthTransactionArgs and SendTxArgs.  Since both implementation has the same methods, `setDefaults` and `toTransaction`, already, you can just add getters or setters for From, Gas, GasPrice, and Nonce

Reply:
i have added EthTransactionArgs in NewTxArgs Interface. 
Other parts of api_ethereum.go can be refactored by these, and it will be reflected in next PR